tests/parser/functions/rlp/test_rlp_list.py

```diff
@@ -64,6 +64,16 @@ def too(inp: bytes <= 100) -> bool:
 def voo(inp: bytes <= 1024) -> num:
     x = RLPList(inp, [num, num, bytes32, num, bytes32, bytes])
     return x[1]
+
+@public
+def loo(inp: bytes <= 1024) -> num:
+    x = RLPList(inp, [num, num, num, num, num, num, num, num, num, num])
+    return x[0] + x[1] + x[2] + x[3] + x[4] + x[5] + x[6] + x[7] + x[8] + x[9]
```

Comment:
Haha, nice. Isn't this a pain coding tho?
``` sum(x[:9])```

Reply:
@OktayGardener Thanks for your suggestions, but Viper doesn't support `sum` or 
`[num] * n

Reply:
Ah, I see, too bad.